app/assets/stylesheets/components/_table.scss

```diff
@@ -0,0 +1,20 @@
+table.dataTable {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `dataTable` should be written in lowercase with hyphens